README.md

```diff
@@ -110,6 +110,41 @@ possible by the use of the `UserService` service provided by
 You can see the details in
 [`packages/shopping/src/controllers/user.controller.ts`](https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/controllers/user.controller.ts).
 
+### Authorization
+
+For an authorization demo in this example, all requests to the endpoint
+`/users/{userId}/orders` (i.e, `UserOrderController`) are validated for specific
+user access.
+
+Endpoint authorization is done using `@loopback/authorization` package and the
+provisions in it are implemented using
+[Casbin](https://github.com/casbin/casbin) configurations.
+
+The following is a typical scenario for an authorization demo:
+
+1. An existing user credentials is used to login using the endpoint
+   `/users/login`. The JWT token returned has the user name (lets say the user's
+   name is `bob`) and the user's id (lets say bob's id is `123`).
```

Comment:
same here for `lets say`

Reply:
I am going to reword this entirely with user bob's actions